app/src/org/commcare/utils/FileUtil.java

```diff
@@ -121,7 +121,7 @@ public static boolean cleanFilePath(String fullPath, String extendedPath) {
 
     private static final String illegalChars = "'*','+'~|<> !?:./\\";
 
-    public static String SanitizeFileName(String input) {
+    public static String sanitizeFileName(String input) {

```

Comment:
think you can ignore lint errors in the code you have not changed as method renames will need to be replicated while calling the method as well.

Reply:
Sorry if I wasn't clear but the file name will need to be reverted here, as otherwise the project fails to compile.